Framework7/src/requests/task_post.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../data/me_details'
+
+class TaskPost
+
+  def self.call_and_return_response_task(end_point,description)
```

Comment:
Think in pass a generic hash instead of a simple description, because if you want to create a task with another field you will need to update this method